dora/core/server/master/src/main/java/alluxio/master/job/DoraLoadJob.java

```diff
@@ -724,16 +708,20 @@ private String getTextReport() {
           progress.append(format("\t\t%s%n", stack.toString()));
         }
       }
+      progress.append(format("\tLoad data: %s%n", mLoadData));
       progress.append(format("\tFiles Processed: %d%n", mProcessedFileCount));
-      progress.append(format("\tBytes Loaded: %s%s%n",
-          FormatUtils.getSizeFromBytes(mLoadedByteCount),
-          mTotalByteCount == null
-              ? "" : format(" out of %s", FormatUtils.getSizeFromBytes(mTotalByteCount))));
-      if (mThroughput != null) {
-        progress.append(format("\tThroughput: %s/s%n",
-            FormatUtils.getSizeFromBytes(mThroughput)));
+      progress.append(format("\tDirectories Processed: %d%n", mProcessedDirectoryCount));
+      if (mLoadData) {
+        progress.append(format("\tBytes Loaded: %s%s%n",
+            FormatUtils.getSizeFromBytes(mLoadedByteCount),
+            mTotalByteCount == null
+                ? "" : format(" out of %s", FormatUtils.getSizeFromBytes(mTotalByteCount))));
+        if (mThroughput != null) {
+          progress.append(format("\tThroughput: %s/s%n",
+              FormatUtils.getSizeFromBytes(mThroughput)));
+        }
+        progress.append(format("\tBlock load failure rate: %.2f%%%n", mFailurePercentage));
       }
-      progress.append(format("\tBlock load failure rate: %.2f%%%n", mFailurePercentage));
       progress.append(format("\tFiles Failed: %s%n", mFailedFileCount));
```

Comment:
nit: better unify the processed/failure here since we may have directory failure. Actually, we can combine the two lines into  `files/directories processed`.